RevenueCatUI/Templates/Components/Text/TextComponentView.swift

```diff
@@ -26,10 +26,13 @@ struct TextComponentView: View {
         Text(viewModel.text)
             .font(viewModel.textStyle)
             .fontWeight(viewModel.fontWeight)
+            .lineLimit(nil)
```

Comment:
I believe this is the default? https://developer.apple.com/documentation/swiftui/environmentvalues/linelimit